.gitignore

```diff
@@ -86,6 +86,7 @@ celerybeat-schedule
 .venv
 venv/
 ENV/
+documenters-aggregator/
```

Comment:
I'm curious if it's wise to use this as the name of the virtualenv. I'd find it confusing myself. But I think it's likely others may name it similarly, so a good idea to put it in the gitignore.